doc/specs/OOBE-Dialog.md

```diff
@@ -0,0 +1,112 @@
+
+Goals:
+
+- Create a guided prompt that exposes the user to a brief overview of each PowerToy, highlighting the PowerToys’ basic usage.
+- Present a short-list of useful customization options that the user can select to define their experience.
+
+Non-goals:
+
+- Present an exhaustive list of features and options that are available in PowerToys. This OOBE prompt is only meant to provide initial exposure for which interested users should continue investigating via provided links to documentation and settings.
+
+## 2. Definition of Success
+
+### 2.1. Customers
+
+The PowerToys OOBE is for new power users and developers who are looking to tune and streamline their Windows experience for greater productivity and enhanced user experience. As the PowerToys customer base tends to be particularly biased against OOBE prompts in general, we need to present the PowerToys OOBE dialog in such a way that it provides immediate value to end-users to improve the likelihood of users discovering all the PowerToys by completing the prompts.
+
+### 2.2. Expected Impact: Customer, and Technology Outcomes
+
+- **High Reliability:** Less than 0.1% crash rate.
+- **Increased Activation:** 30% or more increase in usage in all PowerToys features.
+- **High User Retention:** 25% or more active PowerToys users after 28 days.
+- **High OOBE Engagement Rate** PowerToys users click through at least half of the available OOBE sections.
+
+## 3. Requirements
+
+  The base of this OOBE dialog was attributed by [Niels’s mock-up](https://github.com/microsoft/PowerToys/issues/1285).
+
+### 3.1. Functional Requirements
+
+#### 3.1.1. Functional Requirements Overview
+
+|No. | Requirement | Pri |
+| - | - | - |
+|1 | The OOBE Dialog window should launch immediately after installation. | P0 |
+|2 | The OOBE Dialog should consist of a collection of pages that describe the available PowerToys, organized based on the template depicted in [Figure 5.1.1.](#511-oobe-dialog-layout). | P0 |
+|3 | On launch, present a Welcome/Landing page in the OOBE Dialog window as described in [Section 3.1.3.](#313-welcomeoverview-page-content)  | P1 |
+|4 | Traverse the various OOBE Dialog pages via an list view options on the left-hand pane. | P0 |
+|5 | Alternatively traverse the various OOBE Dialog pages via an on-screen next/previous buttons on each page. | P2 |
+|6 | At any time, the user should be able to exit the OOBE dialog by closing the window. | P0 |
+|7 | Alternatively, the user should be able to exit the OOBE dialog by pressing the ESC key. | P1 |
+|8 | Automatically save changes to settings as they are made in the OOBE. | P1 |
+|9 | In the PowerToys settings “General” menu, add a button to re-launch the OOBE dialog under the “Check for updates” button. | P0 |
+|10 | The window is resizable and the content is responsive to being resized. | P0 |
+|11 | The OOBE dialog should have a preset minimum window size to ensure all content can be viable with the need to scroll the page. | P1 |
```

Comment:
OK, but on small screens that might not be technically possible.

Reply:
Understandable, some scrolling would probably be ok under those cases for extraneous details like the pro-tips section. The point really is just to keep the content as concise as possible to maintain relevance to the general audience. So this requirement is more of a target, not a necessity.